src/nixpacks/app.rs

```diff
@@ -32,6 +35,22 @@ impl App {
         Ok(contents)
     }
 
+    pub fn find_match(&self, re: &Regex, pattern: &str) -> bool {
```

Comment:
Just for documenting, I've created a little discussion over on the Discord about potentially making this function faster. [Link](https://canary.discord.com/channels/713503345364697088/960605685920452669/960669111279550484)